tdei-ui/src/components/ForgotPassModal/ForgotPassModal.js

```diff
@@ -0,0 +1,36 @@
+
+import React, { useState } from 'react';
+import { Modal, Button } from 'react-bootstrap';
+import { Link } from "react-router-dom";
+
+const ForgotPassModal = () => {
+    const [show, setShow] = useState(false);
+    const handleClose = () => setShow(false);
+    const handleShow = () => setShow(true);
+
+    return (
+        <>
+            <div>
+                Forgot Password?{" "}
+                <Link className="tdei-primary-link" onClick={handleShow}>
+                    Click Here
+                </Link>
+            </div>
+            <Modal show={show} onHide={handleClose}>
+                <Modal.Header closeButton>
+                    <Modal.Title>Forgot Password?</Modal.Title>
+                </Modal.Header>
+                <Modal.Body>
+                    <p>Please send an email with your username to : admin@tdei.com</p>
```

Comment:
Please change the email to tdei@uw.edu

Reply:
Changed the email : 
<img width="1470" alt="Screenshot 2023-11-27 at 10 42 25 AM" src="https://github.com/TaskarCenterAtUW/TDEI-user-management-front-end/assets/142565248/20cd67c0-bfbb-4571-915e-720cb78d9fac">